hwcomposer/hwcomposer.pro

```diff
@@ -28,7 +28,7 @@ SOURCES += hwcomposer_backend_v11.cpp
 HEADERS += hwcomposer_backend_v11.h
 
 
-QT += core-private compositor-private gui-private platformsupport-private
```

Comment:
What is this dependency for, actually? When doing a cursory glance around I don't see anything using it, and I would be quite surprised if there was wayland-specific code here.

Even more surprised, given this is apparently all that is required (since 5.7's compositor API is a complete rewrite...)

Can you try remove this, and see what breaks?